### ISSUE TITLE
Rerun with strings.Builder?

### ISSUE THREAD

**Issue by btanner** [hermanschaaf/go-string-concat-benchmarks#8]
Hi. It's a bit of work for me to submit code from work, but I would encourage you to run this again and include strings.Builder in the mix.

From my own test, it is faster than some of the alternatives, and simpler.

**Comment by hermanschaaf** [comment]
Good point, I expect it to be faster too. But haven't had the time to do the test yet. PRs welcome, otherwise I'll update the code and post when I have some time in the future
